lib/Backend/GlobOpt.h

```diff
@@ -318,18 +323,14 @@ template<> ValueNumber JsUtil::ValueToKey<ValueNumber, Value *>::ToKey(Value *co
 
 class IntConstantValueInfo : public ValueInfo
 {
-private:
-    const int32 intValue;
 
-protected:
-    IntConstantValueInfo(const int32 intValue)
-        : ValueInfo(GetInt(IsTaggable(intValue)), ValueStructureKind::IntConstant),
-        intValue(intValue)
+public:
+    static IntConstantValueInfo *New(JitArenaAllocator *const allocator, const int32 intValue)
```

Comment:
The template version initially looked a little bit less complicated & verbose, because I was using inheriting c-tors. Unfortunately, some CI jobs use an older cl compiler (on win7) which gets confused by this feature.